data/achievements.cfg

```diff
@@ -1,5 +1,14 @@
 #textdomain wesnoth
 
+# macros for adding achievement icons
+#define ACHIEVEMENT_ICON_WITH_BASE BASE_IMG ICON_IMG FRAME_IMG
+data/core/images/{BASE_IMG}~SCALE(72,72)~BLIT({ICON_IMG}, 0, 0)~BLIT({FRAME_IMG}, 0, 0)
+#enddef
```

Comment:
The #enddef here and below needs to be on the same line as the contents of the macro.